RobotWeDo/src/main/resources/wedo.properties

```diff
@@ -22,7 +22,8 @@ robot.real.name = WeDo
 robot.vendor = na
 
 # robot's info for further information
-robot.info = https://education.lego.com/de-de/product/wedo-2?CMP=KAC-EDDE17JulWeDo2.0AdwordsDEadwordsWeDo
+robot.info.en = https://education.lego.com/en-us/support/wedo
```

Comment:
Products page does not seem to exist in English, is this a suitable replacement or should we not link to the English site alltogether @bjost2s?